client/src/components/profile/components/Camper.js

```diff
@@ -46,6 +47,26 @@ function joinArray(array) {
   });
 }
 
+function parseDate(joinDate) {
+  const monthNames = [
+    'January',
+    'February',
+    'March',
+    'April',
+    'May',
+    'June',
+    'July',
+    'August',
+    'September',
+    'October',
+    'November',
+    'December'
+  ];
+  const year = joinDate.getFullYear();
+  const month = monthNames[joinDate.getMonth()];
```

Comment:
```suggestion
  // TODO: revisit this when we localise to different languages.
  const month = joinDate.toLocaleDateString('en-US', { month: 'long' });
```

Reply:
@ojeytonwilliams I actually had tried out this way earlier, from here - https://stackoverflow.com/a/18648314

The first code snippet prints `"November"` when you run it in the answer, which is just what we want, but when I tried this with the `ISODate` obtained from MongoDB, the format I got was different - 

![Untitled90](https://user-images.githubusercontent.com/11348778/79037590-88e64e80-7ba0-11ea-8d63-efa3e9f21b14.png)

I didn't want to spend too much time experimenting with it, before getting the main features done, so I just went ahead with the direct approach [specified in the answer](https://stackoverflow.com/a/1643468).

Reply:
The Mongo shell doesn't necessarily use the same JavaScript engine as the browser, so the shell's behaviour is likely to not match the browser's.  However, there's reasonable browser support for [toLocaleString](https://caniuse.com/#search=toLocaleString).  What I had not realised that there was better support for that than `toLocaleDateString`, so I updated my suggestion.

I think the support is high enough that we do need a fallback.  @ahmadabdolsaheb what do you think?

Reply:
> I think the support is high enough that we do need a fallback. @ahmadabdolsaheb what do you think?

Do you mean we would not need a fallback because there is good browser support?

if so, I agree. the iso date should be sent to the client and tested there. If we could get it to work with toLocaleString, I think we should go with it.

Reply:
Yep. Caniuse has 95% support and I assume (maybe you have stats on this) that most people using fcc have reasonably up-to-date browsers.

Reply:
Technically we support only the latest 'LTS' Operating Systems and browsers.